solutions/fscloud/version.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.3.0, <1.6.0"
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "1.62.0"
```

Comment:
Any specific reason that we are using 1.62.0 ? Generally we define range, and here no range is specified.

Reply:
We need to choose the latest version...so check and confirm.

Reply:
1.62.0 is the latest version
https://github.com/IBM-Cloud/terraform-provider-ibm